docs/resources/dms_kafka_instance.md

```diff
@@ -55,21 +95,27 @@ The following arguments are supported:
 * `name` - (Required, String) Specifies the name of the DMS kafka instance. An instance name starts with a letter,
   consists of 4 to 64 characters, and supports only letters, digits, hyphens (-) and underscores (_).
 
-* `product_id` - (Required, String) Specifies a product ID, which includes bandwidth, partition, broker and default
+* `flavor_id` - (Optional, String) Specifies the kafka flavor ID, e.g. **c6.2u4g.cluster**.
+  This parameter and `product_id` are alternative.
+
+  -> The flavor IDs are not supported in some regions.
```

Comment:
pls add the URL of flavor document

Reply:
[Document](https://support.huaweicloud.com/intl/en-us/productdesc-kafka/Kafka-specification.html) has been supported.